tensorflow/contrib/tensorrt/resources/trt_int8_calibrator.cc

```diff
@@ -65,32 +62,32 @@ bool TRTInt8Calibrator::setBatch(
 
     // TODO(aaroey): we should not use sync copy on default stream. Make sure
     // stream->ThenMemcpy() is used in future PRs.
-    auto status =
-        cudaMemcpy(d.first, it.second, d.second, cudaMemcpyDeviceToDevice);
+    auto status = cudaMemcpyAsync(d.first, it.second, d.second,
```

Comment:
Can we use stream->ThenMemcpy() instead? The TF convention is to avoid calling cuda functions directly in kernel code as much as possible (although there are still a few places we cannot avoid), also the wrapper has done a few other things like logging which would be helpful for future debugging.

Reply:
I am not sure on whether we can use it or not. Added a TODO to figure out an alternative